apps/docs/src/pages/docs/layout.mdx

```diff
@@ -1,3 +1,180 @@
 # Layout
 
-This is the Layout page!
+When building UIs you should always strive for **responsive layouts** to enable your pages to adapt to different screen sizes and orientations,
+ensuring overall consistency across platforms and environments.
+
+The NEXT UI Kit provides the following components to help you create responsive layouts:
+
+- [Container](#container)
+- [Grid](#grid)
+- [Simple Grid](#simple-grid)
+
+One underlying concept for all these components are **breakpoints**. While the container and grid components use internally the breakpoints defined by
+the Design System, the simple grid component enables you to create your own breakpoints. These breakpoints are specific screen sizes at which the layouts
+should adjust themselves to suit the screen resolution and orientation.
+
+The breakpoints defined by the Design System are:
+
+| Breakpoint | Width (px) | Gutters (px) | Number of columns |
+| ---------- | ---------- | ------------ | ----------------- |
+| xs         | 0          | 16           | 4                 |
+| sm         | 600        | 16           | 8                 |
+| md         | 960        | 32           | 12                |
+| lg         | 1270       | 32           | 12                |
+| xl         | 1920       | 32           | 12                |
+
+Whenever the screen resolution goes above or below a breakpoint, the layout should adapt itself to suit the screen size and for the content to be accessible.
+
+## Container <a id="container" />
+
+The [`Container`](/components/container) is one of the most basic layout components we provide to our community, and it enables you to center content horizontally on the page.
+
+This component is very useful when your content can't grow beyond a specific breakpoint. In this case, you can use `maxWidth` to specify the breakpoint
+to which the content is bounded. The content will automatically grow or shrink according with the size of the screen and breakpoint used ensuring that
+it doesn't grow beyond the specified width.
+
+If necessary, multiple nested containers can be used as shown in the example below.
+
+```tsx live
+<HvContainer maxWidth="lg">
+  <HvGlobalActions title="Details" />
+  <HvContainer maxWidth="md">
+    <HvTypography variant="body">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti quasi
+      sed perferendis optio eos excepturi possimus atque aperiam impedit
+      expedita dolores, est nulla, aut iure, deleniti recusandae corporis eaque
+      provident?
+    </HvTypography>
+    <HvTypography variant="body">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit culpa error
+      vitae fugit minima. Distinctio tempore ipsa voluptatum, vel alias possimus
+      aut, itaque corrupti nesciunt accusantium commodi nulla minima voluptates.
+    </HvTypography>
+  </HvContainer>
+</HvContainer>
```

Comment:
We could simplify this a bit.
Also, unless we use `smallWidth="sm"`, we can't really see the effect of `HvContainer`
```suggestion
<HvContainer maxWidth="sm">
  <HvGlobalActions title="Details" />
  <HvTypography>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti quasi
    sed perferendis optio eos excepturi possimus atque aperiam impedit
    expedita dolores, est nulla, aut iure, deleniti recusandae corporis eaque
    provident?
    <br />
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit culpa error
    vitae fugit minima. Distinctio tempore ipsa voluptatum, vel alias possimus
    aut, itaque corrupti nesciunt accusantium commodi nulla minima voluptates.
  </HvTypography>
</HvContainer>
```